### PR TITLE
Add writer badge to dropdown on members page

### LINKED ISSUE #3534: [Community] Add the writers program UX badge to the members profile
#### Description
Since we have now have a community badge for the Writer program, it would be great to have it added to the members profile 

#### Page Link:
- [Members](https://layer5.io/community/members/) 
- Badge is available on [Figma ](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=1690%3A10102&t=BDGY3h6mSVmZiBAn-1)

#### Expected Behavior
Add the newly created writers badge to the dropdown on the members profile

#### Screenshots
Screenshot 2022-12-20 at 9 09 54 PM

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by chinu-anand**
Signed-off-by: Chinu Anand <chinuanand81@gmail.com>

**Description**

This PR fixes #3534 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/pages/community/members.js [hunk 1]
```diff
@@ -206,7 +215,7 @@ const MembersPage = () => {
         <Dropdown options={options} handleChange={handleChange} />
 
         {!members ? (
-          <MultipleMembers members={[options[10]]} />
+          <MultipleMembers members={[options[11]]} />
```
Comment (Nikhil-Ladha) [inline src/pages/community/members.js:218]: We need to get away with this fixed number-based approach and better use the array length or some other way such that every time we don't need to update the size.
@chinu-anand do you mind fixing it along in this PR?

**Review by Nikhil-Ladha — COMMENTED** [review]
Also, please add the `SVG` version of the image (along with the PNG version) and use that in the code.
Update the path to `images/writer-program/` under `assets`

**Review by Anita-ihuman — APPROVED** [review]
This looks Great @chinu-anand

**Comment by l5io** [comment]
🚀 Preview for commit 597af6ee68337ed23832bc63e659d9b382155eb3 at: https://63c0ea5d770fe27da1c2701b--layer5.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]
Looking good!

**Comment by Nikhil-Ladha** [comment]
Hey @chinu-anand , while I did merge this PR but I think we missed out on an update in a couple more places like the handbook, and the individual member's page.
Do you mind updating those areas as well in a new PR?
Take a ref from this PR: #3389

**Comment by chinu-anand** [comment]
yeah sure @Nikhil-Ladha, there was issue #3533 already created and people assigned to it, that's why I didn't touch the handbook section. But now as there is no open PR for that, I am going to work on it. Thanks 😄

**Comment by Nikhil-Ladha** [comment]
> yeah sure @Nikhil-Ladha, there was issue #3533 already created and people assigned to it, that's why I didn't touch the handbook section. But now as there is no open PR for that, I am going to work on it. Thanks 😄

Ohh, I didn't know that there is an issue for thaf already.
In that case, instead of touching the handbook just update the individual members page.

**Pull request merged**